packages/elements/src/calendar/__test__/calendar.value.test.js

```diff
@@ -156,6 +188,37 @@ describe('calendar/Value', function () {
       expect(el.value).to.equal('2005-04-01');
     });
 
+    it('It should be possible to select custom cell content on Spacebar', async function () {
+      const el = await fixture(`
+        <ef-calendar view="2005-04" lang="en-GB">
+          <div slot="2005-04-01">
+            <ef-icon icon="cross"></ef-icon>
+          </div>
+        </ef-calendar>
+      `);
+      const cells = getDateCells(el);
+      await keyboardEvent(cells[0], ' ');
+      await keyboardEvent(cells[0], ' ', 'keyup'); // April 01
+      expect(el.value, 'value is not set').to.equal('2005-04-01');
+    });
+
+    it('It should not be possible to deselect custom cell content on Spacebar', async function () {
+      const el = await fixture(`
+        <ef-calendar view="2005-04" lang="en-GB">
+          <div slot="2005-04-01">
+            <ef-icon icon="cross"></ef-icon>
+          </div>
+        </ef-calendar>
+      `);
+      const cells = getDateCells(el);
+      await keyboardEvent(cells[0], ' ');
+      await keyboardEvent(cells[0], ' ', 'keyup'); // April 01
+      expect(el.value, 'value is not set').to.equal('2005-04-01');
+      await keyboardEvent(cells[0], ' ');
+      await keyboardEvent(cells[0], ' ', 'keyup'); // April 01
+      expect(el.value).to.equal('2005-04-01');
+    });
+
```

Comment:
I think slot feature should have unit test more than click or spacebar test. for example dom snapshot etc.

Reply:
I don't think that we should add snapshot for the slot. It generally appears on the calendar as updated snapshots.

Reply:
Okay, make sense. One more thing, how about test the slot click? from my understand unit test just test cell click not slot click.

Reply:
I mean unit test should test `getCellContent` method as well. does cell cover this method use case as well?

Reply:
It's cover.